code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -109,6 +109,8 @@
 	// *** Acid spray *** //
 	///Number of tiles of the acid spray cone extends outward to. Not recommended to go beyond 4.
 	var/acid_spray_range = 0
+	///How long the acid spray stays on floor before it deletes itself.
+	var/acid_spray_duration = 0
```

Comment:
This cannot be 0 as it affects timers.
```suggestion
	var/acid_spray_duration = 1
```

Reply:
reference a minimum of 1 in the autodoc.

Reply:
oh okay